### PR TITLE
feat: app slice

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/common/types/Candidate.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
 import { User } from './User'
 
-export type Candidate = User
+export type Candidate = User & {
+  linkedin: string
+  github: string
+  resume: string
+  position: string
+  country: string
```
Comment (juliswer) [inline src/common/types/Candidate.ts:8]: what about the image?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -25,6 +35,11 @@ export default function Home(): React.ReactElement {
             <code className={styles.code}>pages/index.ts</code>
           </p>
 
+          {JSON.stringify(isLoading)}
```
Comment (juliswer) [inline src/pages/index.tsx:38]: should we have this here?

File: src/pages/index.tsx [hunk 3]
```diff
@@ -25,6 +35,11 @@ export default function Home(): React.ReactElement {
...
+
+          <button onClick={findExample} type="button">
+            findInterviewById
+          </button>
```
Comment (juliswer) [inline src/pages/index.tsx:42]: some than above

**Review by juliswer — APPROVED** [review]
Please check my comments before merging.
On the other side, it looks amazing

File: src/common/types/Candidate.ts (hunk 1 above)
Comment (AlefrankM) [reply src/common/types/Candidate.ts:8]: Is inheritanced from UserType

File: src/pages/index.tsx (hunk 2 above)
Comment (AlefrankM) [reply src/pages/index.tsx:38]: Hahahha

File: src/pages/index.tsx (hunk 3 above)
Comment (AlefrankM) [reply src/pages/index.tsx:42]: XD

**Review by angelstackit — APPROVED** [review]
Good

**Review by Naiby — APPROVED** [review]
Good job

**Pull request merged**